deploy/osd-console-branding/config.yaml

```diff
@@ -0,0 +1,6 @@
+deploymentMode: "SelectorSyncSet"
+selectorSyncSet:
+  matchExpressions:
+  - key: api.openshift.com/product
+    operator: In
+    values: ["osd"]
```

Comment:
```suggestion
  - key: api.openshift.com/product
    operator: NotIn
    values: ["rosa"]
```

Recommend inverse so any new product values are not orphaned.  They'll get OSD branding by default unless/until updated like ROSA.

Reply:
:+1:

Reply:
+1 to this